ansible/roles-infra/infra-vmc-resources/tasks/revoke_vcenter_access.yaml

```diff
@@ -0,0 +1,27 @@
+- name: Get Sandbox Student IPs 
+  uri:
+    url: "{{ nsxt_proxy_url }}/policy/api/v1/infra/domains/mgw/groups/Sandbox_Student_IPs"
+    method: GET
+    headers:
+      csp-auth-token: "{{ _nsxt_token }}"
+    return_content: yes
+  register: _sandbox_public_ips
+
+- name: Set a new variable removing the public IP from the Sandbox Student IPs
+  set_fact: 
+    _sandbox_public_ips_new: "{{ _sandbox_public_ips.json.expression.0.ip_addresses|default([])|difference([vmc_vcenter_student_ip]) }}"
```

Comment:
What if multiple deployments use the same IP? such as at an event where everything is on the same wifi?